vagrant-kubernetes-istio/setup_dockerdaemon_linux.sh

```diff
@@ -3,7 +3,10 @@
 # Setting up docker daemon on host
 echo "Adding insecure registry to docker daemon in host..."
 echo "You old docker daemon file can be found at /lib/systemd/system/docker.service_old"
-sudo cp /lib/systemd/system/docker.service /lib/systemd/system/docker.service_old
+sudo ls /lib/systemd/system/docker.service_old
+if [ $? -ne 0 ]; then
+    sudo cp /lib/systemd/system/docker.service /lib/systemd/system/docker.service_old
+fi
```

Comment:
Do we want to overwrite docker.service_old even it exists? I am thinking maybe this docker.service_old is outdated. If user does not run cleanup_linux_host.sh after testing and a few days later user modify docker daemon. When user run setup_dockerdaemon.sh again, we fail to backup latest docker.service_old

Reply:
How about we give user an option here that an old file exist do you want to overwrite it?

Reply:
good idea!